index.bs

```diff
@@ -325,6 +325,16 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Client-side-resident Credential Private Key</dfn>
+:: A [=Client-side-resident Credential Private Key=] is stored either on the client platform, or in some cases on the authenticator
```

Comment:
My understanding is that the key may *not* always reside in the authenticator. Specifically, the authenticator may encrypt the key to itself, and store the encrypted key in the Credential ID that the RP stores, expecting to receive it back from the RP in the get() call.

For roaming authenticators, storing it in the browser or OS isn't an option, since it needs to be available for another login to the same RP when the same authenticator is attached to a different browser+OS. Having the RP store it with the Credential ID is still viable in that case.

@rlin1, since you weren't already thinking of that model, that makes you really valuable in wording the spec, so that it can explain this option to other newcomers. Can you think of a clear way to specify the requirement that the authenticator *not* use the scheme I just described? That's the intent of setting `requireResidentKey` to true.